app/views/posts/_add_comments.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_with url: "/users/#{@post.author_id}/posts/#{@post.id}/comments", scope: :comment do |form| %>
+    <div class="add-comment-c">
+        <%= form.text_area :text, placeholder: "Add your comment...", rows: 14, required: true, class: "comment-text-c" %>
+        <%= form.submit "Comment", class: "submit-btn" %>
+    </div>
+<% end %>
```

Comment:
- [ ] I think to reach the comment form ı have to type this URL `users/1/posts/1/comments/new` but ı can't even reach this URL `users/1/posts/1` Kindly fix it.
- [ ] I can't check if the like button is working or not because it cant reaches the posting page. Kindly check if the like button working or not after fixing it.
- [ ] Also to reach the form ı have to type the URL on the address bar. It will be better if you add a form link on the page so users can reach fastly. ### OPTIONAL
![image](https://user-images.githubusercontent.com/38635158/192040178-80d2a830-98a9-4d3f-b4e9-008559bf859d.png)